server/app/services/applicant/ReadOnlyApplicantProgramService.java

```diff
@@ -85,6 +85,9 @@ public interface ReadOnlyApplicantProgramService {
   /** Returns if the Program eligibility criteria are met. */
   boolean isApplicationEligible();
 
+  /** Returns if any of the answered questions in the program are not eligible. */
+  boolean isInProgressApplicationNotEligible();
```

Comment:
suggestion (blocking): Just reading the javadoc on this method and comparing to the method above on `isApplicationEligible`, I might guess that this method is just `!isApplicationEligible()`. Please add more detail to the javadoc above and to this one so it's clear how they implement different behavior beyond just inverting the same condition and that it is possible and expected for there to be cases when both return `false`.

Reply:
Updated - let me know if it is more clear. Did a test and saw that applicationIsEligible returns false when a summary page is loaded where the eligibility question hasn't been answered yet and isApplicationNotEligible will return false in that case too.